tests/helper/helper_cli.go

```diff
@@ -26,6 +26,7 @@ type CliRunner interface {
 
 	// HasNamespaceProject returns whether the specified namespace or project exists in the cluster
 	HasNamespaceProject(name string) bool
+	ListNamespaceProject(name string)
```

Comment:
Can you add a comment here that would help understand when to use `HasNamespaceProject` vs. `ListNamespaceProject`?